src/main/resources/openapi/PrintAPIs.yaml

```diff
@@ -666,6 +806,72 @@ components:
         - photoLocation
         - validOnDate
 
+    GenerateAnonymousElectorDocumentRequest:
+      title: GenerateAnonymousElectorDocumentRequest
+      description: Request body to generate an Anonymous Elector Document (AED)
+      type: object
+      properties:
+        gssCode:
+          type: string
+          example: E12345678
+          minLength: 9
+          maxLength: 9
+          description: GSS code of the ERO responsible for the application
+        sourceType:
+          $ref: '#/components/schemas/SourceType'
+        sourceReference:
+          type: string
+          example: 1f0f76a9a66f438b9bb33070
+          description: Reference in the source application that this Temporary Certificate is for.
+        applicationReference:
+          type: string
+          example: V3JSZC4CRH
+          description: The application reference as known by the citizen. Not guaranteed to be unique.
+        electoralNumber:
+          type: string
+          example: V3JSZC4CRH
+          description: The electoral number allocated by the Local Authority to the elector.
+        certificateLanguage:
+          $ref: '#/components/schemas/CertificateLanguage'
+        supportingInfoFormat:
+          $ref: '#/components/schemas/SupportingInformationFormat'
+        photoLocation:
+          type: string
+          description: |
+            The location of the Elector's Certificate photo. 
+            This is the photo that has been prepared and approved for the AED by the vc-admin, rather than the photo that the applicant uploaded as part of their original application. 
```

Comment:
Very minor, but strictly speaking should be
`approved for the AED by the vc-anonymous-admin`
(a vc-admin cannot work on anonymous applications)

Reply:
Done.